tests/pdk/finfet_pdk/test_circuits_partial_routing.py

```diff
@@ -9,6 +9,9 @@
 
 cleanup = False
 
+bypass_errors = False
```

Comment:
@Lastdayends You can set this to True if you want to check that no violations occurs with these partial routing tests.

Reply:
@stevenmburns I see. Thank you.